great_expectations/core/usage_statistics/usage_statistics.py

```diff
@@ -222,15 +222,15 @@ def get_usage_statistics_handler(args_array: list) -> Optional[UsageStatisticsHa
 
 
 # Mapping between method's qualified name and the event name it emits
-# Used to esnure proper usage stats coverage in tests
+# Used to ensure proper usage stats coverage in tests
 ENABLED_METHODS: dict[str, UsageStatsEvents] = {}
 
 
 def usage_statistics_enabled_method(
-    func: Optional[Callable] = None,
-    event_name: Optional[UsageStatsEvents] = None,
-    args_payload_fn: Optional[Callable] = None,
-    result_payload_fn: Optional[Callable] = None,
+    func: Callable | None = None,
+    event_name: UsageStatsEvents | None = None,
+    args_payload_fn: Callable | None = None,
+    result_payload_fn: Callable | None = None,
```

Comment:
Is there anything else about the callable that we can declare here?
Like what they are expected to return or the params they accept?

Reply:
unfortunately its a super variable input list (the fn in question here is any method we decorate)